lib/helpers/link_helper.rb

```diff
@@ -1,5 +1,6 @@
 module LinkHelper
   CANONICAL_ROOT = 'https://www.ably.io/' unless defined?(CANONICAL_ROOT)
+  DOCS_ROOT = 'https://docs.ably.io/' unless defined?(DOCS_ROOT)
```

Comment:
shouldn't this be `.com`?

Reply:
Mm, I was not aware it was migrated given docs.ably.io still works.  @kennethkalmer @jamienewcomb who migrated docs.ably.io to .com?  It needs redirects in place.  I will change to .com Bruce, thanks.

Reply:
See 7439acb

Reply:
We have a ticket to do this. Its with @kennethkalmer https://ably.atlassian.net/browse/FEA-558